test/unit/specs/components/common/PagePreferences.spec.js

```diff
@@ -1,24 +1,29 @@
 import setup from "../../../helpers/vuex-setup"
 import PagePreferences from "renderer/components/common/PagePreferences"
+import lcdClientMock from "renderer/connectors/lcdClientMock.js"
+
 jest.mock(`renderer/google-analytics.js`, () => () => {})
 
 describe(`PagePreferences`, () => {
   let wrapper, store
-  let instance = setup()
+  let { stakingParameters } = lcdClientMock.state
+  let { mount } = setup()
```

Comment:
You know I'm a kind of fan of this :)

Minor: `let` to `const` where possible, there are some instances of those possible changes along the PR

Reply:
I'd prefer to do that refactor on all the test files afterwards along with the `({ wrapper, store })`